ui/selectors/multichain/networks.test.ts

```diff
@@ -268,7 +276,6 @@ describe('Multichain network selectors', () => {
         ...mockState,
         metamask: {
           ...mockState.metamask,
-          solanaSupportEnabled: false,
```

Comment:
Same here, we should probably re-disable Solana support here, to preserve the initial test logic?